bp/index.html

```diff
@@ -4700,6 +4716,95 @@ <h2>Cross reference of use case requirements against best practices</h2>
+<ul><li>
+<a href="https://www.go-fair.org/fair-principles/i1-metadata-use-formal-accessible-shared-broadly-applicable-language-knowledge-representation/">I1. (Meta)data use a formal, accessible, shared, and broadly applicable language for knowledge representation.</a>
+<br/>
+This covers (minimally) data formats, for which see <a href="#semantic-thing">Best Practice 4</a> (<a href="https://www.w3.org/TR/dwbp/#MultipleFormats">DWBP 14</a>), but also commonly used controlled vocabularies and “a good data model”. <i>(Is there a SDWBP/DWBP for this? There is a little bit about vocabs for the geometry, and some of the metadata standards allow reference to vocab & data model)</i>
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/i2-metadata-use-vocabularies-follow-fair-principles/">I2. (Meta)data use vocabularies that follow FAIR principles</a>
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/i3-metadata-include-qualified-references-metadata/">I3. (Meta)data include qualified references to other (meta)data</a> is similar to <a href="#linking">Best Practice 3</a> and <a href="#entity-level-links">Best Practice 10</a>.
+</li></ul>
+
+<h3>Reusable</h3>
+	    <p>"The ultimate goal of FAIR is to optimise the reuse of data. To achieve this, metadata and data should be well-described so that they can be replicated and/or combined in different settings.</p>
+<ul><li>
+<a href="https://www.go-fair.org/fair-principles/r1-metadata-richly-described-plurality-accurate-relevant-attributes/">R1. (Meta)data are richly described with a plurality of accurate and relevant attributes</a>
+<br/>
+See <a href="https://www.w3.org/TR/dwbp/#StructuralMetadata">DWBP Best Practice 3: Provide structural metadata</a>, which doesn’t require the attibution to be 'richly described' or for there to be many attributes (let alone that they be accurate & relevant); but it does at least say that whatever attribution you do provide should be described.
+<br/>
+See <a href="https://www.w3.org/TR/dwbp/#EnrichData">DWBP Best Practice 31: Enrich data by generating new data</a>
+</li>
+<li><a href="https://www.go-fair.org/fair-principles/r1-1-metadata-released-clear-accessible-data-usage-license/">R1.1. (Meta)data are released with a clear and accessible data usage license</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataLicense">DWBP Best Practice 4: Provide data license information</a>; again, this doesn’t say that the license should be "clear", but it does say that it should be accessible - "attached to data" (via a link from the metadata or embedded in the metadata).
+</li><li>
+<a href="https://www.go-fair.org/fair-principles/r1-2-metadata-associated-detailed-provenance/">R1.2. (Meta)data are associated with detailed provenance</a>
+<br/>
+<a href="https://www.w3.org/TR/dwbp/#DataProvenance>DWBP Best Practice 5: Provide data provenance information</a> "Provide complete information about the origins of the data and any changes you have made."
```

Comment:
The DWBP Best Practice 5 does not get rendered when I view this locally. It looks like the `<a>` tag was closed improperly.

Reply:
It was the href string which wasn't closed